core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -149,6 +152,59 @@ boolean testClassLoad(final String className, final String asTypeName)
    */
   void waitForBalance() throws AccumuloException;
 
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param args
+   *          Command line arguments passed in from user command.
+   * @param secretOption
+   *          Specified instance secret to use for commands.
+   *
+   * @since 2.1.0
+   */
+  boolean fateFail(List<String> args, String secretOption) throws AccumuloException;
```

Comment:
I think I agree with @EdColeman 's observation that you can probably just require these to be used with an authenticated user with Admin permission, rather than provide the instance secret. The server side can use the instance secret from its config file, without needing to pass it.

Not sure from this what values are appropriate for "args". Are these just a list of FaTE transaction IDs to fail? The parameters to these methods could be more specific.

Reply:
> Not sure from this what values are appropriate for "args". Are these just a list of FaTE transaction IDs to fail? The parameters to these methods could be more specific.

Yes, Args will be the TiDs. Initially I was going to have it be more broad and include other things inside it. I can see how it is confusing now. Since I have parameters specifically for FilterTiDs for the print statement I could possibly combine the two parameters.

Reply:
Done in my latest commit